platzky/admin/fake_login.py

```
@@ -0,0 +1,53 @@
+    
+    import os
+    if os.environ.get("FLASK_ENV") == "production":
+        import warnings
+        warnings.warn(
+            "Fake login routes are enabled in a production environment! "
+            "This is a serious security risk and should be disabled immediately.",
+            UserWarning, stacklevel=2
+        )
+
+    @blueprint.route("/fake-login/<role>")
+    def handle_fake_login(role: str) -> flask.Response:
```

Ruff (0.8.2)

Undefined name `flask`

(F821)